.changes/README.md

```diff
@@ -0,0 +1,53 @@
+# CHANGELOG Automation
```

Comment:
From what I can tell this file is a copy of what's in `dbt-core`.  Might be worth just linking to that `README` so when we can make changes in a specific place.  If there are differences in adapters, we could call those out specifically in addition to linking to the `README` in `dbt-core`.

Reply:
linked to it, please review and suggest any wording changes.